less/hotgrid.less

```diff
@@ -32,6 +37,25 @@
     .icon-plus;
   }
 
+  &__item-completion-icon {
+    display: none;
+    position: absolute;
+    bottom: 0.5rem;
+    right: 0.5rem;
+    border-radius: 50%;
+    padding: 1rem;
```

Comment:
The icon overlaps the image which could be an issue for some imagery however reducing the padding will mean less of the image is obscured. I think 0.5rem looks good.

e.g `padding: @item-padding / 2`

Reply:
Also, I recall we were querying `.5rem` vs `0.5rem` on a recent PR (can't remember where I saw this). Do you remember if we agreed to either @swashbuck?

Reply:
Personally, I prefer 0.5rem.

Reply:
> Personally, I prefer 0.5rem.

My preference would be 0.5rem as my eyes sometimes miss the decimal 👀

Reply:
@kirsty-hames Yeah, that was on the nav labels ticket. We ended up using an existing variable, but I agree that 0.5rem is less prone to misreading. All in favor of keeping the leading zero.

Reply:
@kirsty-hames Ok, padding reduced to 0.5rem ( @item-padding / 2 )